### DISCUSSION TITLE
Deploy ARM templates into your landing zones - guidance

### DISCUSSION THREAD

**Discussion by MilesCameron-DMs** [Azure/Enterprise-Scale#469] [Q&A] [answered]
Hi guys, i have been reading the document 'Deploy your own ARM templates with AzOps GitHub Actions' but i cant tell from that article if you are suggesting that we should use the existing pipelines (AzOps - Push and Pull) to deploy IaC into the landing zones or if you recommend that we should build our own pipelines at the landing zone or resource group level?

If we are using our own pipelines should we also use separate code repos for our landing zone IaC?

Any guidance would be helpful - thanks

**Comment by krnese** [comment]
For the landing zones, the intention is that app teams bring their own tooling and pipelines into their landing zones

**Reply by MilesCameron-DMs** [reply]
Thanks Kristian. In our case we are both the platform team and we also develop some of the landing zones / resources so i am trying to set out the best way forward in that context.

**Reply by krnese** [reply]
Absolutely - @ljtill FYI :-)

**Reply by MilesCameron-DMs** [reply]
Happy to engage if we can help or provide feedback 👍

**Comment by krnese** [comment, marked as answer]
We recently published the Enterprise-Scale Landing Zones user guide. Can you see if this helps? https://github.com/Azure/Enterprise-Scale/wiki
